og.module

```diff
@@ -303,6 +303,37 @@ function og_entity_type_alter(array &$entity_types) {
   }
 }
 
+/**
+ * Implements hook_theme().
+ */
+function og_theme($existing, $type, $theme, $path) {
+  return [
+    'og_member_count' => [
+      'variables' => [
```

Comment:
Definitely not needed but we _could_ also add the flags for the types we're counting to the template (i.c. blocked pending users).

Reply:
OK this could be handy to generate some CSS classes for group administrators. Will add them to the data that is passed to the theme layer.